src/app/public/modules/ag-grid/ag-grid.service.ts

```diff
@@ -101,10 +107,16 @@ function dateComparator(date1: any, date2: any): number {
  */
 @Injectable()
 export class SkyAgGridService {
+  public currentTheme: SkyThemeSettings;
 
   constructor(
+    @Optional() public themeSvc: SkyThemeService,
     private agGridAdapterService: SkyAgGridAdapterService
-  ) {}
+  ) {
+    if (this.themeSvc) {
+      this.themeSvc.settingsChange.subscribe(settingsChange => this.currentTheme = settingsChange.currentSettings);
```

Comment:
You'll need to unsubscribe to `this.themeSvc.settingsChange` when this component is destroyed, preferably using the `ngUnsubscribe` pattern used in other components. I'd also move this into `ngOnInit()`.